alpine/create-required-so-tar.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -eux
+
+lib_files=()
+usr_lib_files=()
+usr_local_lib_files=()
+
+for line in $(ldd /usr/local/lib/postgresql/pgroonga.so 2>/dev/null); do
+  if [ "${line:0:1}" != "/" ]; then
+    continue
+  fi
```

Comment:
いやぁ、これは乱暴じゃない？

こうとか？

```suggestion
for line in $(ldd /usr/local/lib/postgresql/pgroonga.so 2>/dev/null | grep -E -o '(/[^ ]+)'); do
```